File: ArrowMove/src/game/system/GameProperties.java
package game.system;

import java.util.Properties;

public class GameProperties extends Properties {
	
	
	public int getPropertyInt(String key) {
		String pr = super.getProperty(key);
		if (pr == null) return -1;
		return Integer.parseInt(pr);
	}
	
	public int getPropertyInt(String key,int defaultValue) {
		String pr = super.getProperty(key);
		if (pr == null) return defaultValue;
		return Integer.parseInt(pr);
	}

}

File: ArrowMove/src/game/system/GameProperties.kt
package game.system

import java.util.Properties

class GameProperties : Properties() {


    fun getPropertyInt(key: String): Int {
        val pr = super.getProperty(key) ?: return -1
        return Integer.parseInt(pr)
    }

    fun getPropertyInt(key: String, defaultValue: Int): Int {
        val pr = super.getProperty(key) ?: return defaultValue
        return Integer.parseInt(pr)
    }

}
